jupyter_server/services/kernels/kernelmanager.py

```diff
@@ -121,8 +121,10 @@ def _default_kernel_buffers(self):
         help="The last activity on any kernel, including shutting down a kernel")
 
     def __init__(self, **kwargs):
-        super(MappingKernelManager, self).__init__(**kwargs)
+        self.pinned_superclass = MultiKernelManager
+        self.pinned_superclass.__init__(self, **kwargs)
         self.last_kernel_activity = utcnow()
+        self.log.info("Kernel manager started")
```

Comment:
Since we're now logging in serverapp that async kernel management is in use and this was the default behavior anyway, I'm not sure how useful this is.